src/controllers/lessonsController.js

```diff
@@ -133,6 +133,14 @@ const createLesson = async (req, res) => {
   const createdBy = req.user.userId;
   const userRole = req.user.role;
   const { studentId } = req.params;
+  const {
+    lessonTitle,
+    lessonDescription,
```

Comment:
later in this code you have repetitive. Please check which part of code you want o leave
<img width="735" alt="image" src="https://github.com/user-attachments/assets/a3f3af01-c0df-465a-9a45-14bc6d185735">

Reply:
Fixed